css-two-column-layout/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full item">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quaerat molestiae modi tenetur aperiam iusto dolore autem pariatur. Quibusdam, omnis ad!</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-half row feature">
```

Comment:
No one element should be both a `row` and a `column`. You should make a child element that has the `row` class inside this `column-one-half` element.